examples/api/demo_affine_image.py

```diff
@@ -24,22 +21,15 @@ def get_image():
     return Z
 
 
-def imshow_affine(ax, z, *kl, **kwargs):
-    im = ax.imshow(z, *kl, **kwargs)
-    x1, x2, y1, y2 = im.get_extent()
-    im._image_skew_coordinate = (x2, y1)
```

Comment:
sigh... did we seriously have this private member documented in our examples? That kinda sends mixed messages to our users, doesn't it?